scenes/Enemy.tscn

```diff
@@ -16,11 +16,11 @@ extents = Vector2( 32, 32 )
 [node name="Enemy" type="KinematicBody2D"]
 
 input_pickable = true
-collision_layer = 1
-collision_mask = 1
+collision_layer = 2
+collision_mask = 4
```

Comment:
Hmmm, is the mask on `KinematicBody2D` really needed? The player is killed (or takes damage) when the enemy's `Area2D` reports a collision... I guess this really depends on game design and whether we want to the player to be able to pass through monsters (taking damage in the process) or not.

In both cases it should be redundant with the current setup.

Reply:
My bad 🙏

Reply:
I actually meant the layer, not the mask.